src/themes/money/theme.json

```diff
@@ -3292,6 +3292,59 @@
               "compounds.card.variants.horizontal.headerChildren"
             ]
           }
+        },
+        "assuranceBar": {
+          "display": "flex",
+          "flexDirection": "row",
+          "flexWrap": "wrap",
+          "alignItems": "flex-start",
+          "textAlign": "left",
+          "py": 0,
+          "mb": "md",
+          "h3": {
+            "color": "#000",
+            "paddingLeft": "0px",
+            "fontFamily": "Nunito, sans-serif",
+            "fontWeight": 600,
+            "fontSize": 30,
+            "lineHeight": "120%"
+          },
+          "p": {
+            "color": "#000",
+            "mx": 0,
+            "fontFamily": "SF Pro Text",
```

Comment:
not sure where this font has come from, but it should not be used on money
should be fontFamily: "base"

Reply:
It's the font used in designs :/